x/ibc/20-transfer/types/msgs.go

```diff
@@ -0,0 +1,63 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	TypeMsgTransfer = "transfer"
+)
+
+type MsgTransfer struct {
+	SrcPort      string         `json:"src_port" yaml:"src_port"`
+	SrcChannel   string         `json:"src_channel" yaml:"src_channel"`
+	Denomination string         `json:"denomination" yaml:"denomination"`
+	Amount       sdk.Int        `json:"amount" yaml:"amount"`
+	Sender       sdk.AccAddress `json:"sender" yaml:"sender"`
+	Receiver     string         `json:"receiver" yaml:"receiver"`
```

Comment:
shouldn't this be an address too?

Reply:
The address bech32 prefix may be different in different chains, seems string is better.

Reply:
yeah but chains must define their own prefix tho